src/llfs/packed_array.hpp

```diff
@@ -116,6 +121,22 @@ struct PackedArray {
       out << "} PackedArray;" << std::endl;
     };
   }
+
+  void initialize_size_in_bytes(usize size_in_bytes)
+  {
+    this->size_in_bytes = size_in_bytes;
+    flags = static_cast<Flags>(flags | kSizeInBytesSet);
+  }
+
+  bool is_valid_size_in_bytes() const
```

Comment:
nit: the phrasing here is slightly awkward; consider using `has_size_in_bytes()` instead.

Reply:
To me 'has_size_in_bytes' is more abstract. When we read 'is_valid_XX' is clearly says what is that we are trying to do (in this case check against). I see that we used 'has_XXX' elsewhere  so will go with that. I will change it.

Reply:
I can see your point in a general sense; but I do feel that when choosing amongst different alternative naming patterns to represent a specific idea, it's probably a bit better to use one that makes grammatical sense in the most cases.  At any rate, you're right that there is an existing convention; thanks for making the change!